### DISCUSSION TITLE
GitHub Free Storage limit

### DISCUSSION THREAD

**Discussion by pradkuma007** [community/community#22830] [New to GitHub]
<p>Greetings !</p>
<p>As per the documentations, GitHub Free offers unlimited collaborators in private repos and 500MB package storage.</p>
<p>From my account, I have created muliple repositories.</p>
<p>Now this 500 MB storage limit is for all repositories owned by my account or individual repository will have a 500 MB storage ?</p>
<p>Thanks in advance.</p>
<p>Prad</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/pradkuma007">@pradkuma007</a>,</p>
<p>Thank you for being here! That allowance is per account not repository. </p>

**Comment by pradkuma007** [comment]
<p>Thank you, much appreciated the details sahred !</p>

**Comment by alkanna** [comment]
<p>Hey <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>, bumping this because I am wondering if it’s per team account or github user account?<br>
Meaning, if we have 10 people in our team, is it 500MB storage per user or 500MB for the whole team?</p>
<p>Thanks.</p>
